site/content/usage/16-fargate-support.md

```diff
@@ -0,0 +1,301 @@
+> cluster` before creating the Fargate profile.
+
+```console
+$ eksctl create fargateprofile --namespace dev --cluster fargate-example-cluster
+[ℹ]  creating Fargate profile "fp-9bfc77ad" on EKS cluster "fargate-example-cluster"
+[ℹ]  created Fargate profile "fp-9bfc77ad" on EKS cluster "fargate-example-cluster"
+```
+
+
+You can also specify the name of the Fargate profile to be created. This name must not start with the prefix `eks-`.
+
+```console
+$ eksctl create fargateprofile --name eks-dev --namespace dev --cluster fargate-example-cluster --name fp-development
+[ℹ]  created Fargate profile "fp-development" on EKS cluster "fargate-example-cluster"
+```
+To see existing Fargate profiles in a cluster:
+
+```console
+$ eksctl get fargateprofile --cluster fargate-example-cluster
+NAME		POD_EXECUTION_ROLE_ARN										SUBNETS										SELECTOR_NAMESPACE	SELECTOR_LABELS
+fp-9bfc77ad	arn:aws:iam::123456789012:role/eksctl-fargate-example-cluster-ServiceRole-1T5F78E5FSH79	subnet-00adf1d8c99f83381,subnet-04affb163ffab17d4,subnet-035b34379d5ef5473	dev		<none>
+```
+And to see them in `yaml` format:
+
+```console
+$ eksctl get fargateprofile --cluster fargate-example-cluster -o yaml
+- name: fp-9bfc77ad
+  podExecutionRoleARN: arn:aws:iam::123456789012:role/eksctl-fargate-example-cluster-ServiceRole-1T5F78E5FSH79
+  selectors:
+  - namespace: dev
+  subnets:
+  - subnet-00adf1d8c99f83381
+  - subnet-04affb163ffab17d4
+  - subnet-035b34379d5ef5473
+```
+
+Or in `json` format:
+
+```console
+$ eksctl get fargateprofile --cluster fargate-example-cluster -o json
+[
+    {
+        "name": "fp-9bfc77ad",
+        "podExecutionRoleARN": "arn:aws:iam::123456789012:role/eksctl-fargate-example-cluster-ServiceRole-1T5F78E5FSH79",
+        "selectors": [
+            {
+                "namespace": "dev"
+            }
+        ],
+        "subnets": [
+            "subnet-00adf1d8c99f83381",
+            "subnet-04affb163ffab17d4",
+            "subnet-035b34379d5ef5473"
+        ]
+    }
+]
+```
+
+Fargate profiles are immutable by design. To change something, create a new Fargate profile with the desired changes and
+delete the old one with the `eksctl delete fargateprofile` command like in the following example:
+
+
+```console
+$ eksctl delete fargateprofile --cluster fargate-example-cluster --name fp-9bfc77ad --wait
+2019-11-27T19:04:26+09:00 [ℹ]  deleting Fargate profile "fp-9bfc77ad"
+  ClusterName: "fargate-example-cluster",
+  FargateProfileName: "fp-9bfc77ad"
+}
+```
+
+Note that the profile deletion is a process that can take up to a few minutes. When the `--wait` flag is not specified,
+`eksctl` optimistically expects the profile to be deleted and returns as soon as the aws request has been sent. To make
+`eksctl` wait until it has been successfully deleted use `--wait` like in the example above.
+
+### Further reading
+
+#<!-- TODO: Add link to AWS EKS Fargate docs -->
```

Comment:
Where is this documentation published? Can we update this `TODO`?

Reply:
Presumably https://docs.aws.amazon.com/eks/latest/userguide/fargate.html, will open a follow-up PR

Reply:
See #1627